src/test/groovy/org/gradle/android/ZipMergingTaskWorkaroundTest.groovy

```diff
@@ -0,0 +1,51 @@
+package org.gradle.android
+
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.TaskOutcome
+import spock.lang.Unroll
+
+@MultiVersionTest
+class ZipMergingTaskWorkaroundTest extends AbstractTest {
```

Comment:
Thanks for adding the test. I wonder if we can add additional tasks to execute to the existing `CrossVersionOutcomeAndRelocationTest`. @ghale what do you think?

Reply:
Yep didn't want to mess up the existing `CrossVersionOutcomeAndRelocationTest` but if we have more tasks not in the assemble task graph, it would probably be better to incorporate them there.

Reply:
Yeah, this feels like something that should go into `CrossVersionOutcomeAndRelocationTest`.  We have dedicated tasks for workarounds that have some sort of special behavior, but since this one just enables/disables caching, I don't think we need a separate test.  Am I understanding correctly that this task is not captured by `assemble`?  If so, then I'd say let's either add the `createFullJar` tasks to the command line or change the `SimpleAndroidApp` build scripts to add these as dependencies to `assemble`.

Reply:
You're absolutely right @ghale, I'll add the task as a dependency to `assemble` to make this smooth.
great suggestion 👍